awx/main/tests/unit/test_tasks.py

```diff
@@ -61,7 +61,10 @@ def patch_Job():
 
 @pytest.fixture
 def job():
-    return Job(pk=1, id=1, project=Project(), inventory=Inventory(), job_template=JobTemplate(id=1, name='foo'))
+    return Job(
+        pk=1, id=1,
+        project=Project(local_path='/projects/_23_foo'),
```

Comment:
Again, `local_path` _has_ to be set for all non-manual projects. This is an oddball case because it's an object not even saved to the database, so our normal means of setting it won't work.